.codecov.yml

```diff
@@ -0,0 +1,6 @@
+ignore:
```

Comment:
Here are some additional config options for a quieter experience with Codecov:

```yaml
# Don't post a comment on pull requests.
comment: off

# Disable commit statuses
coverage:
  status:
    project: no
    patch: no
    changes: no
```

via https://github.com/JanusGraph/janusgraph/blob/master/codecov.yml

Reply:
Thanks!